### PR TITLE
Windows build support

### PR DESCRIPTION
**Pull request by maxhora**

### CODE REVIEW

**Review by xhochy — APPROVED** [review]
+1, cool!

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_BUILD_TYPE:STRING=%CMAKE_CONFIG% ^
+      -DBUILD_SHARED_LIBS:BOOL=ON ^
+      "%SRC_DIR%"
+nmake
+ctest -V
+
+move bro.exe "%LIBRARY_BIN%\"
+move *.dll "%LIBRARY_BIN%\"
```
Comment (xhochy) [inline recipe/bld.bat:14]: This line confuses me a bit, but this seems to be difference on Windows that DLLs and Executables live in the same folder?

Comment (maxhora) [reply recipe/bld.bat:14]: Yes, usually .exe and .dll files are placed in bin directory; All .lib files are placed in lib. That's common for Windows.
Others conda-forge's feedstocks are doing similarly (for instance, curl's exe is copied to bin; boost's dlls are copied to bin too ).

Comment (maxhora) [reply recipe/bld.bat:14]: It seems that the core reason of this difference, is in how run-time dependency loading mechanism is working on Windows. Dependence Loader searches dependencies using PATH variable, dependencies paths are not hard-coded in binaries. So it makes sense to keep .dlls and .exes in the same folder added to the PATH.

**Comment by wesm** [comment]
Brotli 0.6 has just been released -- the presence of a Python 2.7 wheel for Windows suggest maybe the Windows issues have been sorted out? https://github.com/google/brotli/releases

**Pull request merged**